api/src/tool/console/interactive.py

```diff
@@ -120,7 +120,7 @@ def reload_repl() -> None:
 
         try:
             importlib.reload(module)
-        except:  # noqa: B001
+        except Exception:
```

Comment:
This is the only necessary code change. We used to ignore bare exceptions, but I think I prefer having that check in place. Since it only occurred in a single place within a local development tool and the docs even explicitly state that one reason you don't want to do this is keyboard interrupts, I think its just better to get rid of it:
https://docs.astral.sh/ruff/rules/bare-except/